src/cadenza.js

```diff
@@ -108,6 +108,15 @@ globalThis.cadenza = Object.assign(
  * * `"workbook-view-management"` - Add/Edit/Remove workbook views (Is included in 'workbook-design'.)
  * */
 
+/**
+ * @typedef Distance
+ * @property {number} value
+ * @property {LengthUnit} lengthUnit
+ */
+/**
+ * @typedef {'m'|'km'} LengthUnit
+ */
```

Comment:
Should this be "Buffer"?
```javascript
/**
 * @typedef Buffer
 * @property {number} value
 * @property {'m'|'km'} unit
 */
```
Should we make the unit optional (assuming "m" as the default)?

Reply:
optional is "* @property {'m'|'km'} [unit] ", but how to set a default value on the property?

Reply:
This should work:
```
 * @property {'m' | 'km'} [unit='m']
```